### ISSUE TITLE
Pressing Esc while voice search is open should close the UI

### ISSUE THREAD

**Issue by sevaan** [mozilla/speaktome#161]
There is no keyboard shortcut to dismiss the Voice Fill UI. The Esc key should do this.

**Comment by SoftVision-CarmenFat** [comment]
Hi @andrenatal

I've tried verifying this fix on the latest Voice Fill build (built on 9/14/2017) and here are the results:
	
1. On Windows 10 x64 and Ubuntu 16.04 x64, pressing the ESC key **during the recording process** didn't have any effect; the Voice Fill overlay was dismissed by the ESC key only when the **multiple search results** were displayed.
	**_Also, I noticed that the recording overlay can be dismissed through ESC key only after starting a new voice search by clicking the "Reset" button. Should I file a new issue for this?_**
	
2. On Mac 10.12, the ESC key dismisses the overlay on both occasions: while the recording process took place and when the multiple answers were displayed.

**Comment by andrenatal** [comment]
@SoftVision-CarmenFat yes, please. File new issues specifying you are encountering problems on Windows and Linux and that the behavior differ from Mac

**Comment by andrenatal** [comment]
Actually, let's keep it simple and just reopen this one.

**Comment by andrenatal** [comment]
Fix on:
- [ ] Mac
- [ ] Windows
- [ ] Linux

**Comment by andrenatal** [comment]
@SoftVision-CarmenFat I just pushed a new fix. Could you please test this again and reopen if you still reproduce it?

**Comment by SoftVision-CarmenFat** [comment]
Tested this again on Windows and Linux and the issue is no longer reproducible on the pre-release Voice Fill v.1.3.1.
